packages/env/lib/init-config.js

```diff
@@ -55,3 +64,18 @@ module.exports = async function initConfig( { spinner, debug } ) {
 
 	return config;
 };
+
+function dockerFileContents( image, xDebugMode ) {
```

Comment:
I think it's helpful to have our own Dockerfile at this point. I imagine this could open up the path to some other features too.